components/badge/index.css

```diff
@@ -185,275 +84,270 @@
+		background: var(--spectrum-green-background-color-default);
+	}
 
-.spectrum-Badge--cyan {
-	background: var(--mod-badge-background-color-cyan, var(--spectrum-badge-background-color-cyan));
-}
+	&:where(.spectrum-Badge--seafoam) {
+		background: var(--spectrum-seafoam-background-color-default);
+	}
 
-.spectrum-Badge--blue {
-	background: var(--mod-badge-background-color-blue, var(--spectrum-badge-background-color-blue));
-}
+	&:where(.spectrum-Badge--cyan) {
+		background: var(--spectrum-cyan-background-color-default);
+	}
 
-.spectrum-Badge--indigo {
-	background: var(--mod-badge-background-color-indigo, var(--spectrum-badge-background-color-indigo));
-}
+	&:where(.spectrum-Badge--blue) {
+		background: var(--spectrum-blue-background-color-default);
+	}
 
-.spectrum-Badge--purple {
-	background: var(--mod-badge-background-color-purple, var(--spectrum-badge-background-color-purple));
-}
+	&:where(.spectrum-Badge--indigo) {
+		background: var(--spectrum-indigo-background-color-default);
+	}
 
-.spectrum-Badge--fuchsia {
-	background: var(--mod-badge-background-color-fuchsia, var(--spectrum-badge-background-color-fuchsia));
-}
+	&:where(.spectrum-Badge--purple) {
+		background: var(--spectrum-purple-background-color-default);
+	}
 
-.spectrum-Badge--magenta {
-	background: var(--mod-badge-background-color-magenta, var(--spectrum-badge-background-color-magenta));
-}
+	&:where(.spectrum-Badge--fuchsia) {
+		background: var(--spectrum-fuchsia-background-color-default);
+	}
 
-.spectrum-Badge--pink {
-	background: var(--mod-badge-background-color-pink, var(--spectrum-badge-background-color-pink));
-}
+	&:where(.spectrum-Badge--magenta) {
+		background: var(--spectrum-magenta-background-color-default);
+	}
 
-.spectrum-Badge--turquoise {
-	background: var(--mod-badge-background-color-turquoise, var(--spectrum-badge-background-color-turquoise));
-}
+	&:where(.spectrum-Badge--pink) {
+		background: var(--spectrum-pink-background-color-default);
+	}
 
-.spectrum-Badge--brown {
-	background: var(--mod-badge-background-color-brown, var(--spectrum-badge-background-color-brown));
-}
+	&:where(.spectrum-Badge--turquoise) {
+		background: var(--spectrum-turquoise-background-color-default);
+	}
 
-.spectrum-Badge--cinnamon {
-	background: var(--mod-badge-background-color-cinnamon, var(--spectrum-badge-background-color-cinnamon));
-}
+	&:where(.spectrum-Badge--brown) {
+		background: var(--spectrum-brown-background-color-default);
+	}
 
-.spectrum-Badge--silver {
-	background: var(--mod-badge-background-color-silver, var(--spectrum-badge-background-color-silver));
+	&:where(.spectrum-Badge--cinnamon) {
+		background: var(--spectrum-cinnamon-background-color-default);
+	}
+
+	&:where(.spectrum-Badge--silver) {
+		background: var(--spectrum-silver-background-color-default);
+	}
 }
 
 /* Style = subtle */
-.spectrum-Badge--style-subtle {
-	color: var(--mod-badge-subtle-label-icon-color, var(--spectrum-badge-subtle-label-icon-color));
+.spectrum-Badge--subtle {
```

Comment:
I'm on board with this change (and I see it's in barebones already 🎉 ), and given that our changeset will reflect alll the other component changes, I think I can see the rationale for _not_ documenting this change in the changeset and I'm on board with that too... but I still want to highlight it for consideration in case someone else has a strong opinion or we have second thoughts.

Reply:
Yes, I did check our changesets to see if there were any references to the new class name and since it wasn't there, I opted to leave it out of the new docs. The concept of what was added is in the previous changeset but not the class name specifically.